### PR TITLE
New package: aquatone-1.4.3

### PR DESCRIPTION
**Pull request by misuchiru03**

### CODE REVIEW

**Comment by maxice8** [comment]
The commit email and your account don't match, are you sure that is desirable ?

File: srcpkgs/aquatone/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'aquatone'
+pkgname=aquatone
+version=1.4.3
+revision=1
+build_style=go
+hostmakedepends="git"
+go_import_path=github.com/michenriksen/aquatone
```
Comment (maxice8) [inline srcpkgs/aquatone/template]: go_import_path above hostmakdepends below build_style

Comment (misuchiru03) [reply srcpkgs/aquatone/template]: Fixed that. I was pasting a line too low using `dd` `p` in vim.

**Comment by misuchiru03** [comment]
> 
> 
> The commit email and your account don't match, are you sure that is desirable ?

Added the commit email to my account.  Really, it's just gmail address tagging for sorting. Not a big problem. Same account.  Does that cause an issue here? If so I can address it.

**Comment by maxice8** [comment]
On Tue, Feb 26, 2019 at 09:32:32AM -0800, misuchiru03 wrote:
> > 
> > 
> > The commit email and your account don't match, are you sure that is desirable ?
> 
> Added the commit email to my account.  Really, it's just gmail address tagging for sorting. Not a big problem. Same account.  Does that cause an issue here? If so I can address it.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9197#issuecomment-467535200

Not an issue but your account won't get credited.

**Comment by misuchiru03** [comment]
Travis failed at aarch64-musl.  Builds locally just fine.

**Pull request merged**